src/tests/TextWidgetShowCase.tsx

```diff
@@ -0,0 +1,6 @@
+import ShowCaseWidgetProps from '../types/Widgets';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export default function TextWidgetShowCase(_props: ShowCaseWidgetProps) {
```

Comment:
```suggestion
export default function TextWidgetShowCase() {
```

Reply:
I need the props, it's in them that i store the widgetName & type